pkg/commands/drift/driftignore.go

```diff
@@ -46,43 +46,50 @@ var ignoredFolderPattern = regexp.MustCompile(`^\/organizations\/(?:\d*)\/folder
-	regexp.MustCompile(`containerthreatdetection.serviceAgent\/serviceAccount:service-(?:\d*)@gcp-sa-ktd-control.iam.gserviceaccount.com`),
-	regexp.MustCompile(`dataflow.serviceAgent\/serviceAccount:service-(?:\d*)@dataflow-service-producer-prod.iam.gserviceaccount.com`),
-	regexp.MustCompile(`editor\/serviceAccount:(?:\d*)-compute@developer.gserviceaccount.com`),
-	regexp.MustCompile(`editor\/serviceAccount:(?:\d*)@cloudservices.gserviceaccount.com`),
-	regexp.MustCompile(`file.serviceAgent\/serviceAccount:service-(?:\d*)@cloud-filer.iam.gserviceaccount.com`),
-	regexp.MustCompile(`firebase.managementServiceAgent\/serviceAccount:firebase-service-account@firebase-sa-management.iam.gserviceaccount.com`),
-	regexp.MustCompile(`firebase.managementServiceAgent\/serviceAccount:service-(?:\d*)@gcp-sa-firebase.iam.gserviceaccount.com`),
-	regexp.MustCompile(`firebaserules.system\/serviceAccount:service-(?:\d*)@firebase-rules.iam.gserviceaccount.com`),
-	regexp.MustCompile(`iap.settingsAdmin\/serviceAccount:(?:\d*)-compute@developer.gserviceaccount.com`),
-	regexp.MustCompile(`identitytoolkit.viewer\/serviceAccount:(?:\d*)-compute@developer.gserviceaccount.com`),
-	regexp.MustCompile(`networkmanagement.serviceAgent\/serviceAccount:service-(?:\d*)@gcp-sa-networkmanagement.iam.gserviceaccount.com`),
-	regexp.MustCompile(`osconfig.serviceAgent\/serviceAccount:service-(?:\d*)@gcp-sa-osconfig.iam.gserviceaccount.com`),
-	regexp.MustCompile(`pubsub.serviceAgent\/serviceAccount:service-(?:\d*)@gcp-sa-pubsub.iam.gserviceaccount.com`),
-	regexp.MustCompile(`redis.serviceAgent\/serviceAccount:service-(?:\d*)@cloud-redis.iam.gserviceaccount.com`),
-	regexp.MustCompile(`run.serviceAgent\/serviceAccount:service-(?:\d*)@serverless-robot-prod.iam.gserviceaccount.com`),
-	regexp.MustCompile(`securitycenter.serviceAgent\/serviceAccount:service-org-(?:\d*)@security-center-api.iam.gserviceaccount.com`),
-	regexp.MustCompile(`securitycenter.serviceAgent\/serviceAccount:service-project-(?:\d*)@security-center-api.iam.gserviceaccount.com`),
-	regexp.MustCompile(`servicenetworking.serviceAgent\/serviceAccount:service-(?:\d*)@service-networking.iam.gserviceaccount.com`),
-	regexp.MustCompile(`storage.admin\/serviceAccount:cloud-data-pipeline@koi-b2637a0100e14f34c8c1-tp.iam.gserviceaccount.com`),
-	regexp.MustCompile(`storage.admin\/serviceAccount:project-(?:\d*)@storage-transfer-service.iam.gserviceaccount.com`),
-	regexp.MustCompile(`storage.objectViewer\/serviceAccount:project-(?:\d*)@storage-transfer-service.iam.gserviceaccount.com`),
-	regexp.MustCompile(`vpcaccess.serviceAgent\/serviceAccount:service-(?:\d*)@gcp-sa-vpcaccess.iam.gserviceaccount.com`),
-	regexp.MustCompile(`websecurityscanner.serviceAgent\/serviceAccount:service-(?:\d*)@gcp-sa-websecurityscanner.iam.gserviceaccount.com`),
+	regexp.MustCompile(`aiplatform\.serviceAgent\/serviceAccount:service-(?:\d*)@gcp-sa-aiplatform\.iam\.gserviceaccount\.com`),
```

Comment:
This LGTM. You could also use https://pkg.go.dev/regexp#QuoteMeta if you don't like the hassle of manually escaping all the special characters.

Reply:
Ack. Will leave as the full escaped pattern for now. Its easier to copy/paste and confirm the pattern is valid.